### ISSUE TITLE
Page component - remove `<aside>` from nav section

### ISSUE THREAD

**Issue by jgiardino** [patternfly/patternfly-react#1136]
A few updates are needed to improve accessibility of the Page component.

- The `<aside>` element should be changed to a generic `<div>`.
  Both `<aside>` and `<nav>` are [landmark regions](https://www.w3.org/TR/WCAG20-TECHS/ARIA11.html) that serve different purposes. The `<nav>` in the page component should not be wrapped in an `<aside>`, but instead should just be a generic `<div>` element.
- `aria-controls` should be added to `.pf-c-page__header-brand-toggle > .pf-c-button` 
  - This change requires an `id` added to `.pf-c-page__sidebar`, with the `aria-controls` value set to this `id` value
- `aria-expanded="true/false"` should be added to `.pf-c-page__header-brand-toggle > .pf-c-button`
- `aria-label` value for the element `.pf-c-page__header-brand-toggle > .pf-c-button` should be changed to "Toggle global navigation"

These updates with additional explanation are included in issues for core: 
- https://github.com/patternfly/patternfly-next/issues/1185
- https://github.com/patternfly/patternfly-next/issues/1184
- https://github.com/patternfly/patternfly-next/issues/1186
- https://github.com/patternfly/patternfly-next/issues/1127
